theano/gpuarray/tests/test_dnn.py

```diff
@@ -26,6 +26,10 @@
 
 from theano.configdefaults import SUPPORTED_DNN_CONV_ALGO_FWD
 
+mode_with_gpu = mode_with_gpu.including()
+# Globally disabled for mode_without_gpu
+mode_with_gpu.check_py_code = False
```

Comment:
I don't like doing it globally. I'm pretty sure many tests rely on that to check the correspondance of both code. But maybe doing the test in FAST_COMPILE as we do do enough, but I'm not 100% sure of that.